tests/Integration/ParserTest.php

```diff
@@ -66,7 +67,7 @@ public function testParseFile()
                     } catch (Exception $e) {
                         if (
                             'Secured pdf file are currently not supported.' !== $e->getMessage()
-                            && 0 != strpos($e->getMessage(), 'TCPDF_PARSER')
+                            && 0 !== strpos($e->getMessage(), 'TCPDF_PARSER')
```

Comment:
please double check this. looked like a bug to me, because the "Missing catalog"-Exception was not catched by this test.

Reply:
The `TCPPDF_PARSER` reference is obsolete there and could be removed. But it should be done separately as long as it doesn't interfere with your fix.

Reply:
So you mean i should remove it in another PR?

Reply:
As long as it doesn't interfere with your fix, leave it be. We can take care of it later on.